dev/util.h

```diff
@@ -78,90 +115,96 @@ namespace sqlite_orm {
             std::function<void(serialize_arg_type sql)> will_run_query;
             std::function<void(serialize_arg_type sql)> did_run_query;
 
-            inline void perform_void_exec(sqlite3* db, orm_gsl::czstring sql) const {
+            void perform_void_exec(sqlite3* db, orm_gsl::czstring sql) const {
```

Comment:
removing `inline` may cause symbol duplicate errors during compilation cause non-template non-inline funcs better be located at `.cpp` files not in headers. Same with other inlines within this file

Reply:
This rule doesn't apply to member functions defined in the class body as they are inlined by default.

Reply:
I skipped that it is located inside class. Ok then